app/views/provider_interface/application_choices/index.html.erb

```diff
@@ -3,16 +3,8 @@
 <% if FeatureFlag.active?('provider_information_banner') %>
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-full">
-      <div class="app-banner" aria-labelledby="change-to-working-days-message">
-        <div class="app-banner__message">
-          <h2 class="govuk-heading-m" id="change-to-working-days-message">
-            20 December 2020 to 1 January 2021 do not count as working days
-          </h2>
-          <p class="govuk-body">
-            The Christmas period will not be included when calculating dates for applications to be automatically rejected. This is to match the dates used by UCAS.
-          </p>
-        </div>
-      </div>
+      <%= render NotificationMessageComponent.new(:info, message: '20 December 2020 to 1 January 2021 do not count as working days',
```

Comment:
Just a note that ERB Lint trips up on indention like this, but fine to leave as for now.

Reply:
I'll take a look. There seem to be a couple of complains as well that I should address. Hoping to get some time so we can investigate how to make this run a bit faster or in parallel as you suggested on the PR.